webpack.example-list.js

```diff
@@ -0,0 +1,73 @@
+const { Example } = require("./webpack-utils/Example");
+
+const examples = [
+    new Example('third-person-camera'),
+    new Example('abstract-cube'),
+    new Example('baked-lighting'),
+    new Example('block-fade'),
+    new Example('bobs'),
+    new Example('bump-map'),
+    new Example('bunny'),
+    new Example('cinematic-scroller'),
+    new Example('cube-tunnel'),
+    new Example('cube'),
+    new Example('cube-subpixel'),
+    new Example('demo'),
+    new Example('distorted-sphere'),
+    new Example('dof-balls'),
+    new Example('fire'),
+    new Example('flood-fill'),
+    new Example('frustum-culling'),
+    new Example('gears-2'),
+    new Example('gears'),
+    new Example('hoodlum'),
+    new Example('led-plasma'),
+    new Example('lens'),
+    new Example('md2'),
+    new Example('mdl'),
+    new Example('metaballs'),
+    new Example('metalheadz'),
+    new Example('misc'),
+    new Example('mode-7'),
+    new Example('moving-torus'),
+    new Example('other-md2', './src/examples/different-md2/Application.ts'),
```

Comment:
why is this line different?

Reply:
directory in the examples folder does not match the entry point name